### ISSUE TITLE
Documentation - Accessibility

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#101718]
### Description:

The accessibility files I have created since joining the team have been saved in the team's Google Drive. They need to be moved over to Github & SharePoint. 

Review
- [x] Review the team's accessibility docs.
- [x] Decide if they should be moved over & where they ought to go - Sharepoint vs GitHub. Not all the files will make sense to move over - team training videos for example. 

Move/copy
- [x] Move files over

Write
- [x] Identify gaps in documentation - See VA Forms team repo for ideas of what should be covered
- [x] Write up missing docs - might need another ticket, not sure how much I'll need to write

**Comment by bacitracin** [comment]
Files moved over from Google Drive -> Sharepoint & Github

Wrote additional documentation on audit methodology & the team's accessibility initiatives [GitHub](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/disability/526ez/accessibility)
